src/init/mod.rs

```diff
@@ -212,6 +219,11 @@ pub fn init_main(shell_name: &str) -> io::Result<()> {
         "powershell" => print_script(PWSH_INIT, &starship_path.sprint()?),
         "ion" => print_script(ION_INIT, &starship_path.sprint()?),
         "elvish" => print_script(ELVISH_INIT, &starship_path.sprint_posix()?),
+        // print_script() adds quotes around starship_path which breaks starship.tcsh
+        "tcsh" => print!(
+            "{}",
+            TCSH_INIT.replace("::STARSHIP::", &starship_path.sprint_posix()?)
```

Comment:
I think you should still add something here to properly escape the path to starship.

Reply:
fixed, it is now consistent with the other shells.